en/qgep_swmm/Import-Results.rst

```diff
@@ -3,11 +3,25 @@
 SWMM Import Results
 ===================
 
+If you want to store the results in QGEP and use the dedicated view for the mapping then use SWMM Import Results.
+It parses the report file of SWMM. 
+The results are imported in the measurements tables in the ``qgep_od`` schema:
+
+- ``measurement_result``, 
+- ``measurement_series``, 
+- ``measuring_device`` ,
+- ``measuring_point`` .
+  
+and they populate a view dedicated for the mapping (see :ref:`mapping-imported`).
+
+.. figure:: images/qgep_od_measurement_tables.jpg
+
 Launch ``SWMM Import results``:
 
 - ``SWMM report file``: the ``.rpt`` file generated during the previous step
 - ``Database``: The name of the PostgreSQL service hosting the QGEP database.
-- ``Simulation name``: The name that you want to give to this simulation.
+- ``Simulation name``: The name that you want to give to this simulation (it will appear in ``qgep_od.measuring_point.remark``).
```

Comment:
@tproduit And what defines the identifier of the connected measurement_series?

Reply:
@tproduit Maybe a sample screenshot of the measurement tables with data could be helpful to understand what is added where